sdk/samples/router/create_router_with_pyfunc_ensembler.py

```diff
@@ -55,7 +55,10 @@ def ensemble(
         if "version" in input:
             return predictions[input["version"]]["data"]["value"]
         elif "Version" in kwargs["headers"]:
-            return predictions[kwargs["headers"]["Version"]]["data"]["value"]
+            if hash(kwargs["headers"]["Turing-Req-Id"]) % 2 == 0:  # If the hash of the turing request id is even
+                return predictions[kwargs["headers"]["Version"]]["data"]["value"]
+            else:
+                return 0
```

Comment:
Added this additional if/else condition to demonstrate how the `Turing-Req-Id` header can be accessed in a Pyfunc ensembler.